### PR TITLE
Fetch translations before build

### PR DESCRIPTION
**Pull request by b-dabrowski**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,9 +5,10 @@
   "scripts": {
     "test": "mocha --exit",
     "start": "node -r dotenv/config  ./dist/server.js",
-    "build": "tsc",
+    "build": "npm run getTranslations && tsc",
     "watch": "nodemon",
-    "local": "node -r dotenv/config ./localtunnel.js"
+    "local": "node -r dotenv/config ./localtunnel.js",
+    "getTranslations": "node -r dotenv/config fetchAndSaveTranslations.js"
```
Comment (TomaszChmielPagepro) [inline package.json]: npm command should be named in `kebab-case`

File: example.env [hunk 2]
```diff
@@ -4,3 +4,5 @@ DB_CONNECTION_STRING=mongodb://localhost:27017/openkudos
 DB_NAME=kudos_test
 CLIENT_ID=1111111111111.11111111111
 CLIENT_SECRET=jk4k44j5j454345k43kkkj5433j4k5j3k
+POE_API_TOKEN=jk4k44j5j454345k43kkkj5433j4k5j3k
+POE_PROJECT_ID=111111
```
Comment (TomaszChmielPagepro) [inline example.env]: I think we can commit real project id ;)

Comment (b-dabrowski) [reply example.env]: It is an example file so other devs will know with of env are necessary to run the project

**Pull request merged**